apps/portals/cancercomplexity/src/config/routesConfig.tsx

```diff
@@ -235,6 +236,10 @@ const routes: RouteObject[] = [
         path: 'MC2Supplement',
         element: <MC2Supplement />,
       },
+      {
+        path: 'Search',
```

Comment:
Unlike the DetailPage tabs, we are going to run the query associated to all tab data for the resulting query count reported on the tab